src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -157,23 +154,32 @@ const BaseComment = (props: BaseCommentProps) => {
         onSelect(false);
     };
 
-    const handleMessageUpdate = ({
-        id: messageID,
-        text,
-        hasMention,
-    }: {
-        hasMention: boolean,
-        id: string,
-        text: string,
-    }): void => {
-        onEdit(messageID, text, undefined, hasMention, permissions);
+    const handleMessageUpdate = ({ id: messageID, text, hasMention }) => {
+        // Since we have to pass onCommentEdit through annotations (to Replies), onAnnotationEdit essentially overrides onCommentEdit
+        if (onAnnotationEdit) {
+            onAnnotationEdit({ id: messageID, text, permissions });
```

Comment:
Annotations can also have mentions in them. Why does onCommentEdit have that flag passed but onAnnotationEdit doesn't. Might be a good reason but I just want to make sure we are aware that annotations have mentions too.

Reply:
I don't know. Aside from converting the multiple params into a single param with props, I didn't change the `onAnnotationEdit` API. The props correspond to the pre-existing params. I do think we should look into if/how annotations previously handled mentions, probably as part of the tech spike.